fission/src/ui/modals/configuring/SettingsModal.tsx

```diff
@@ -9,23 +9,14 @@ import Slider from "@/components/Slider"
 import Checkbox from "@/components/Checkbox"
 import PreferencesSystem from "@/systems/preferences/PreferencesSystem"
 import { SceneOverlayEvent, SceneOverlayEventKey } from "@/ui/components/SceneOverlayEvents"
+import { QualitySetting } from "@/systems/preferences/PreferenceTypes"
+import { Box } from "@mui/material"
 
-const moveElementToTop = (arr: string[], element: string | undefined) => {
-    if (element == undefined) {
-        return arr
-    }
-
-    arr = arr.includes(element) ? [element, ...arr.filter(item => item !== element)] : arr
-    return arr
-}
-
-const screenModeOptions = ["Windowed", "Fullscreen"]
-const qualitySettingsOptions = ["Low", "Medium", "High", "Ultra"]
+const qualitySettingsOptions: QualitySetting[] = ["Low", "Medium", "High"]
```

Comment:
I feel like this and the QualitySettings type on line 32 of PreferenceTypes is kinda redundant. Would it be better to change it?